src/foam/u2/view/FObjectView.js

```diff
@@ -71,6 +71,12 @@ foam.CLASS({
       class: 'Class',
       name: 'of'
     },
+    {
+      class: 'Boolean',
+      name: 'skipBaseClass',
+      documentation: 'If true, skips of base-class as a choice when no strategies fetched.',
```

Comment:
Can you expand on this to say when/why you would want to do this?

Reply:
When we are adding a bank account to a contact, but the user has yet to unlock any corridor strategies, FObjectView sets the choices to the base class of BankAccount.js. In this case it doesn't make sense to have the base class as an option, since we only allow the user to add subclasses of BankAccount.